framework/include/kernels/Reaction.h

```diff
@@ -9,18 +9,29 @@
 
 #pragma once
 
-#include "Kernel.h"
+#include "GenericKernel.h"
 
-class Reaction : public Kernel
+/**
+ *  Implements a simple consuming reaction term with weak form $(\\psi_i, \\lambda u_h)$.
+ */
+
+template <bool is_ad>
```

Comment:
is doxygen ok with a space here?

Reply:
I am not sure. Might as well remove it to be safe.